modules/infinite_datalayer/infinite_datalayer.module

```diff
@@ -95,6 +95,15 @@ function infinite_datalayer_infinite_datalayer_node_alter(&$datalayer_variables,
     }
   }
 
+  if($node->hasField('field_sponsor_type')){
+    if ($node->field_sponsor_type->value) {
+      $datalayer_variables_page['sponsorType'] = $node->field_sponsor_type->value;
+    }
+    else {
+      $datalayer_variables_page['sponsorType'] = 'default';
+    }
+  }
+
```

Comment:
please check and fix your code style settings (space before { on line 98, no new line should be on 101)

Reply:
I added the missing spaces in l. 98, however, Drupal coding standards say "new line before else"